sysvar/src/lib.rs

```diff
@@ -145,10 +142,25 @@ pub fn is_sysvar_id(id: &Pubkey) -> bool {
     ALL_IDS.iter().any(|key| key == id)
 }
 
+/// Interface for loading a sysvar.
+pub trait SysvarGet: Sized {
```

Comment:
I think this is missing a `Default` constraint since the `get` macro implementation is doing:
```rust
let mut var = Self::default();
```